Apps/WebClient/src/ClientApp/src/views/publicCovidTest.vue

```diff
@@ -467,9 +474,16 @@ export default class PublicCovidTestView extends Vue {
                                 data-testid="error-text-description"
                                 class="pl-4"
                             >
-                                {{
-                                    publicCovidTestResponseResultError.description
-                                }}
+                                <p :class="haveErrorDetails ? 'mb-1' : 'mb-0'">
```

Comment:
This works well for adding spacing, but I think the default amount we use between paragraphs is 1rem.  You can achieve this by replacing 'mb-1' with 'mb-3' or by only setting the margin when the details are not present:

`:class="{ 'mb-0': haveErrorDetails }"`